plugins/com.google.cloud.tools.eclipse.projectselector/META-INF/MANIFEST.MF

```diff
@@ -40,6 +40,7 @@ Import-Package: com.google.api.client.auth.oauth2,
  org.eclipse.jface.dialogs,
  org.eclipse.jface.layout,
  org.eclipse.jface.viewers,
+ org.eclipse.jface.window,
```

Comment:
This is a `projectselector` bundle, where the only change in the PR is some SWT flags. I think you put `Window.open()` in this bundle but later refactored it to the `.deploy.ui` bundle?